tao/tao_bridge/ops/fake_quant_op.cc

```diff
@@ -58,7 +47,6 @@ class DiscFakeQuantOp : public OpKernel {
   bool signed_;
```

Comment:
nit: maybe better to used the same name as the attribute name?

Reply:
done